# represents all YOLO detections in one frame

std_msgs/Header header
yolo_ros2_following/Detection[] detections